src/components/VHeader/VSearchBar/VSearchBar.vue

```diff
@@ -211,6 +211,11 @@ export default defineComponent({
         // If a recent search is selected, populate its value into the input.
         modelMedium.value = entries.value[selectedIdx.value]
 
+      // Hide the recent searches popover when the user presses shift+tab on the input.
+      if (key === keycodes.Tab && event.shiftKey) {
+        handleBlur()
```

Comment:
I'm a little bothered by the difference between this line and line 221 below.

I'm not sure if we should use `isRecentVisible.value = false` to match the code in the next if block or update that to use `handleBlur()` instead.

Reply:
I haven't noticed it before, and I'm all in for consistency! I wonder if it's best to extract a `hideRecent` function that would set `isRecentVisible.value` to `false` to make it easier to understand the code.

Reply:
I've renamed `handleBlur` to `hideRecentSearches`, and `handleFocus` to `showRecentSearches` (they actually just set the `isRecentVisible.value` to `false`/`true`), @dhruvkb. I feel like that makes it easier to understand the template code.